### DISCUSSION TITLE
App non si collega, ma via browser è ok

### DISCUSSION THREAD

**Discussion by xmxbx73** [indomus/forum#1652] [HUB: Home Assistant]
Ciao a tutti,
Da qualche settimana non riesco più a collegarmi via app ad home assistant ( impossibile connettersi a home assistant) installato su Raspberry Pi. La cosa strana e che via browser funziona.
Qual è la differenza?
Non so più dove sbattere la testa.
Grazie a chi tenterà di aiutarmi.
Marco.

**Comment by KC-inDomus** [comment]
Marco ci devi raccontare di più. se usi ssl con un fqdn, se la locale ti colleghi ecc

**Comment by xmxbx73** [comment]
Ops, scusa...hai ragione.
Ho configurato un fqdn, ma per sicueezza sto usando un ip fisso pubblico.
Sto usando una connessione https
Anche in locale il problema è lo stesso. App non funziona, da browser tutto ok

**Reply by snakuzzo** [reply]
IP fisso o dinamico non importa.
Come hai configurato l'app? Doppia url interna per quando sei in LAN ed esterna o solo esterna (con fqdn)?
Qial è l'errore vedi?

**Comment by xmxbx73** [comment]
Doppia url interna e esterna con indirizzo ip.
L'errore è 'Impossibile connettersi a Home Assistant. Si è verificato un errore nel caricamento di Home Assistant, rivedi le impostazioni di connessione e riprova...'

**Reply by snakuzzo** [reply]
Dall'esterno devi collegarti con FQDN, non con indirizzo IP perché l'app controlla che sia esposto un certificato rilasciato da una CA valida...ed il certificato è associato al FQDN, non all'indirizzo IP

**Comment by xmxbx73** [comment]
Se utilizzo FQDN non mi funziona neppure via Browser. Che sia un problema di certificato?

**Reply by snakuzzo** [reply]
eh...potrebbe essere tutto o niente...
di sicuro hai sbagliato tu a fare qualcosa.

- Potrebbe essere il certificato sbagliato o proprio non presente
- Potresti aver sbagliato la configurazione lato HA
- Potresti aver sbagliato il port forwarding
- Potrebbe non essere corretta l'associazione ip <-> FQDN

Insomma...dovresti verificare tutte queste cose...non basta dire "va" oppure "non va"

**Comment by xmxbx73** [comment]
Beh...ha smesso di funzionare all'improvviso, senza aver toccato qualcosa...
Quindi il problema non dovrebbe essere un'errata impostazione da parte mia.

**Reply by snakuzzo** [reply]
di sicuro non è che l'applicazione smette di funzionare così per caso.
Quindi controlla quello che ti ho detto e, per cortesia, fai reply all'ultimo messaggio senza aprire nuovi thread (come hai fatto ora)
